README.md

```diff
@@ -1,2 +1,233 @@
-# microverse-hello
-First project - "Hello Microverse"
+<a name="readme-top"></a>
+
+<!---->
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Hello README File</b></h3>
+
+</div>
+
```

Comment:
- [ ] Please consider Updating the Microverse logo with yours if possible or remove it totally. The template is designed by Microverse for us all to use to professionally document our projects and has it filled with content related to each project we may be working on.

Reply:
Good day, Tadesse: I made all changes but I got confused in git with the branches and commands PUSH and MERGE. I think I accidentally merged from the main branch, which caused the Pull Request to close.
Can you explain to me what to do and how should I proceed (make corrections in the feature branch, then commit in that branch and then pull changes to the remote repo from the feature branch? Do I need to merge the main branch in my local repo?). Thank you.

Reply:
Dear @jcmunav63. I believe you can restore and reopen the branch. then you can reset your main branch to the earlier version of it.